db/migrate/20150818181427_update_tenant_divisible_on_existing_rows.rb

```diff
@@ -1,7 +1,8 @@
 class UpdateTenantDivisibleOnExistingRows < ActiveRecord::Migration
   class Tenant < ActiveRecord::Base
   end
```

Comment:
Be sure to add the sti type_disabled clause here that we add everywhere else...it keeps us safe from any future changes to Tenant table.